azurelinuxagent/ga/send_telemetry_events.py

```diff
@@ -92,55 +98,65 @@ def enqueue_event(self, event):
 
         # Queue.put() can block if the queue is full which can be an uninterruptible wait. Blocking for a max of
         # SendTelemetryEventsHandler._MAX_TIMEOUT seconds and raising a ServiceStoppedError to retry later.
+
+        # Todo: Queue.put() will only raise a Full exception if a maxsize is set for the Queue. Once some size
+        # limitations are set for the Queue, ensure to handle that correctly here.
         try:
             self._queue.put(event, timeout=SendTelemetryEventsHandler._MAX_TIMEOUT)
-        except Full as error:
+        except Exception as error:
             raise ServiceStoppedError(
-                "Queue full, stopping any more enqueuing until the next run. {0}".format(ustr(error)))
+                "Unable to enqueue due to: {0}, stopping any more enqueuing until the next run".format(ustr(error)))
+
+    def _wait_for_event_in_queue(self):
```

Comment:
Since None is so special, let's add a comment here describing when this function returns None

Reply:
Done